server/routes/auth.js

```diff
@@ -47,10 +46,11 @@ router.post('/register', validInfo, async (req, res) => {
 
     // Return JWT token
     const jwtToken = jwtGenerator(userId)
-    return res.json({ jwtToken })
+    return res.json({ ok: true, message: 'Registration successful', jwtToken })
   } catch (err) {
+    console.log('in our register error catch')
```

Comment:
Remove all added console.logs before merge, not that I'm the best about this but as a practice it's good to adhere to

Reply:
Ya my bad man I went back and removed these in the next commit, should have just thrown them on here by pushing those but I gotchu